asylum-connect/control-panel
# Grouping tests support

## Description

This PR does what was done in the Catalog Repo, grouping Automation tests into groups. Thus greatly reducing the time for for PRs to pass.

## Asana ticket: https://app.asana.com/0/1132189118126148/1201784448169497

## PR Checklist


## How to Test

Check the Circle CI Checks. The tests are now running in 4 separate groups.

NOTE: Depends on PR  https://github.com/asylum-connect/control-panel/pull/78
